.travis.yml

```diff
@@ -33,7 +33,7 @@ before_install:
   - sudo service mysql stop || sudo service mysql-5.6 stop || sudo stop mysql-5.6 || sudo stop mysql || echo "failed to stop mysql"
   - sudo docker pull mysql:$MYSQL_VERSION
   - sudo docker run --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=root -d mysql:$MYSQL_VERSION
-  - sleep 20
```

Comment:
I think this sleep is needed to make the tests run reliably. Or did you so something here that makes the sleep obsolete?

Reply:
That was an accident sorry, removed